src/test/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/csv/CSVFileReaderTest.java

```diff
@@ -411,7 +411,7 @@ public void testBrokenCSV() {
         }
         try (BufferedInputStream stream = new BufferedInputStream(
                 new FileInputStream(brokenFile))) {
-            new CSVFileReader(new CSVFileReaderSpi()).read(stream, null);
+            new CSVFileReader(new CSVFileReaderSpi(), ',').read(stream, null);
```

Comment:
@oscardssmith is there a test for a TSV file somewhere in here? I don't see it. If not, why not?

Reply:
The main reason is that I felt that a unit-test wouldn't be that useful because most of the difficulty is just in making the ingest happen. I could change one of the csv files to a tsv one though, it's not a bad idea.